sherpa/astro/data.py

```diff
@@ -4701,6 +4699,42 @@ def __init__(self, name, x0lo, x1lo, x0hi, x1hi, y, shape=None,
     def _init_data_space(self, filter, *data):
         return IntegratedDataSpace2D(filter, *data)
 
+    def get_x0(self, filter=False):
+        indep = self._data_space.get(filter)
+        return (indep.x0lo + indep.x0hi) / 2.0
+
+    def get_x1(self, filter=False):
+        indep = self._data_space.get(filter)
+        return (indep.x1lo + indep.x1hi) / 2.0
+
+    @property
```

Comment:
coverage claims that line 4710 was not covered by the tests, which seems a tad odd if line 4711 is covered.